docs/Welcome.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to the WealthKernel Guides
 
-## Api Basics
+## API Basics

```

Comment:
I already changed this in my PR: https://github.com/wealthkernel/api-documentation-guides/pull/42/files#diff-9c127a8af417fbccedfdb9e143929c02fd0c9ee4656950d702ec5f3060b38450R3